host/libcontainer_backend.go

```diff
@@ -389,17 +397,47 @@ func (l *LibcontainerBackend) Run(job *host.Job, runConfig *RunConfig, rateLimit
 		}
 	}()
 
-	log.Info("pulling image")
-	artifactURI, err := l.resolveDiscoverdURI(job.ImageArtifact.URI)
+	imageID, err := pinkerton.ImageID(job.ImageArtifact.URI)
 	if err != nil {
-		log.Error("error resolving artifact URI", "err", err)
+		log.Error("error getting artifact image ID", "err", err)
 		return err
 	}
-	// TODO(lmars): stream pull progress (maybe to the app log?)
-	imageID, err := l.pinkerton.PullDocker(artifactURI, ioutil.Discard)
-	if err != nil {
-		log.Error("error pulling image", "err", err)
-		return err
+
+	var rootPath string
+	switch job.ImageArtifact.Type {
+	case host.ArtifactTypeDocker:
+		log.Info("pulling image")
```

Comment:
I assume the plan is to delete this code entirely? We should translate any images into Flynn format before running them.